bezier-rs/lib/src/lib.rs

```diff
@@ -331,6 +331,67 @@ impl Bezier {
 		};
 		bezier_starting_at_t1.split(adjusted_t2)[t2_split_side]
 	}
+
+	/// Find the roots of the linear equation ax + b
+	fn solve_linear(a: f64, b: f64) -> Vec<f64> {
```

Comment:
Now that https://github.com/GraphiteEditor/Graphite/pull/687 has been merged, you can moves this and `solve_quadratic` into the `utils.rs` file.